packages/storybook-utils/src/style.css

```diff
@@ -0,0 +1,161 @@
+.onyx-disclaimer {
+  border-radius: 0.5rem;
+  padding: 1rem;
+  line-height: 1.5rem;
+  margin: 1rem 0;
+
+  /* same color as VitePress var(--vp-c-tip-soft) */
+  background-color: color-mix(in srgb, var(--onyx-color-base-info-400) 25%, transparent);
+}
+
+.onyx-disclaimer__title {
+  font-weight: 600;
+  margin-bottom: 0.5rem;
+}
+
+.onyx-disclaimer p {
+  margin: 0;
+}
+
+/* the Storybook table of content headline is always black so we need to manually set it for the dark mode */
+#storybook-docs .sbdocs-wrapper > div:nth-child(2) {
+  /* same as Storybook color "textMuted" inside ./theme.ts */
+  color: var(--onyx-color-text-icons-neutral-medium);
+}
+
+/* To prevent bg flashing when changing between elements in darkmode */
+.sb-preparing-story,
+.sb-preparing-docs {
+  background-color: transparent;
+}
+/* removed placeholder for the same reason */
+.sb-previewBlock,
+.sb-argstableBlock {
+  display: none;
+}
+
+/* 
+  Copy of ../sit-onyx/src/styles/variables/themes/onyx.css
+  TODO: Find a way to automate this
+ */
+:where(:root),
```

Comment:
Hmm I don't really like this copy...what do you think about just replacing / hard-coding the two variables in line 8 and 23 since we only need those two values?

Reply:
Oh, you‘re right